src/styles/main.scss

```diff
@@ -1,3 +1,355 @@
-body {
+/* #region page */
+.page {
+  color: #292929;
+  font-family: Space Grotesk, sans-serif;
```

Comment:
You use this `font-family` several times, it is better to transfer it to a variable and reuse it.